### PR TITLE
SD-4208 Update Amazon S3 proxy URL to include version

### PR DESCRIPTION
**Pull request by mugurrus**

### CODE REVIEW

**Comment by akintolga** [comment]
:+1:

**Comment by actionless** [comment]
mb only better smth like `AMAZON_PROXY_VERSION`

File: superdesk/storage/amazon/amazon_media_storage.py [hunk 1]
```diff
@@ -61,7 +61,8 @@ def url_for_media_default(app, media_id, content_type=None):
     extension = str(_guess_extension(content_type)) if content_type else ''
     url = '%s.%s/%s%s' % (app.config['AMAZON_CONTAINER_NAME'], endpoint, media_id, extension)
```
Comment (actionless) [inline superdesk/storage/amazon/amazon_media_storage.py]: extension won't work without proxy so you have to add it under `if app.config.get('AMAZON_PROXY_SERVER'):`

Comment (actionless) [inline superdesk/storage/amazon/amazon_media_storage.py]: or that extension should be added when uploading, so it won't be added by proxy

**Pull request closed without merging**